qiskit_nature/problems/second_quantization/lattice/lattice_model_problem.py

```diff
@@ -0,0 +1,125 @@
+        """
+        eigenstate_result = None
+        if isinstance(raw_result, EigenstateResult):
+            eigenstate_result = raw_result
+        elif isinstance(raw_result, EigensolverResult):
+            eigenstate_result = EigenstateResult()
+            eigenstate_result.raw_result = raw_result
+            eigenstate_result.eigenenergies = raw_result.eigenvalues
+            eigenstate_result.eigenstates = raw_result.eigenstates
+            eigenstate_result.aux_operator_eigenvalues = raw_result.aux_operator_eigenvalues
+        elif isinstance(raw_result, MinimumEigensolverResult):
+            eigenstate_result = EigenstateResult()
+            eigenstate_result.raw_result = raw_result
+            eigenstate_result.eigenenergies = np.asarray([raw_result.eigenvalue])
+            eigenstate_result.eigenstates = [raw_result.eigenstate]
+            eigenstate_result.aux_operator_eigenvalues = [raw_result.aux_operator_eigenvalues]
+        result = LatticeModelResult()
+        result.combine(eigenstate_result)
+        result.computed_lattice_energies = eigenstate_result.eigenenergies
+        return result
+
+    def hopping_qeom_ops(
+        self,
+        qubit_converter: QubitConverter,
+        excitations: Union[
+            str,
+            int,
+            List[int],
+            Callable[[int, Tuple[int, int]], List[Tuple[Tuple[int, ...], Tuple[int, ...]]]],
+        ] = "sd",
+    ) -> None:
+        """Generates the hopping operators and their commutativity information
+        for the specified set of excitations. Raises `NotImplementedError` for the
+        `LatticeProblemModel` class, currently.
+
+        Args:
+            qubit_converter: the `QubitConverter` to use for mapping and symmetry reduction. The
+                             Z2 symmetries stored in this instance are the basis for the
+                             commutativity information returned by this method.
+            excitations: the types of excitations to consider. The simple cases for this input are
+
+                :`str`: containing any of the following characters: `s`, `d`, `t` or `q`.
+                :`int`: a single, positive integer denoting the excitation type (1 == `s`, etc.).
+                :`List[int]`: a list of positive integers.
+                :`Callable`: a function which is used to generate the excitations.
+                    For more details on how to write such a function refer to one of the default
+                    methods, :meth:`generate_fermionic_excitations` or
+                    :meth:`generate_vibrational_excitations`.
+
+        Returns:
+            Currently, this function is not implemented in the `LtticeProblemModel` class and
+            always raises `NotImplementedError`.
+        """
+        raise NotImplementedError(
+            "Currently, it's not implemented in the `LatticeProblemModel` class."
+        )
+
+    def get_default_filter_criterion(
+        self,
+    ) -> None:
```

Comment:
Same here: I would try to keep the function definition aligned with the base class